Apps/Playground/Scripts/experience.js

```diff
@@ -51,6 +53,10 @@ function CreateInputHandling(scene) {
     });
 }
 
+if (chromeDevTools.supportsInspector()) {
+    chromeDevTools.startInspector(5643, "experience.js");
+}
```

Comment:
I'm not sure how I feel about this. I kind of think this code should be in native.

Reply:
Do you mean in `_native` or in `App.cpp`?

Reply:
Latter.